src/docs/Foundations/MagicUnit/code.mdx

```diff
@@ -134,11 +134,11 @@ It basically just do the `to-rem()` part for you
 }
 ```
 
-### The shorthands variables :
+### The shorthands variables
 
-> the **shorthands variables** behave exactly as the magic-unit-rem, but in a variable form factor.
+> The **shorthands variables** behave exactly as the magic-unit-rem, but in a variable form factor.
 
-### The magic unit grid development utility :
+### The magic unit grid development utility
 
 In order to help you to position your elements as well as possible according to the magic-unit, Mozaic provides the css class `mdu-magic-unit-grid` to add on your container and which allows the display of a positioning grid.
```

Comment:
In order to help you to position your elements according to the magic-unit, Mozaic provides the css class `mdu-magic-unit-grid` to apply on your container and shows a positioning grid.

Reply:
fixed

Reply:
fixed